klayout/drc/rule_decks/main.drc

```diff
@@ -14,880 +14,766 @@
 # limitations under the License.
 ################################################################################################
 
-#===========================================================================================================================
-#------------------------------------------- GF 0.18um MCU DRC RULE DECK --------------------------------------------------
-#===========================================================================================================================
+#=======================================================================================================================
+#--------------------------------------------- GF 0.18um MCU DRC RULE DECK ---------------------------------------------
+#=======================================================================================================================
 require 'time'
-require "logger"
+require 'logger'
 require 'etc'
 
 exec_start_time = Time.now
 
-logger = Logger.new(STDOUT)
+logger = Logger.new($stdout)
 
-logger.formatter = proc do |severity, datetime, progname, msg|
-  "#{datetime}: Memory Usage (" + `pmap #{Process.pid} | tail -1`[10,40].strip + ") : #{msg}
+logger.formatter = proc do |_severity, datetime, _progname, msg|
+  "#{datetime}: Memory Usage (" + `pmap #{Process.pid} | tail -1`[10, 40].strip + ") : #{msg}
 "
 end
 
 #================================================
 #----------------- FILE SETUP -------------------
 #================================================
-logger.info("Starting running GF180MCU Klayout DRC runset on %s" % [$input])
-logger.info("Ruby Version for klayout: %s" % [RUBY_VERSION])
+logger.info(format('Starting running GF180MCU Klayout DRC runset on %s', $input))
+logger.info(format('Ruby Version for klayout: %s', RUBY_VERSION))
 
 if $input
-    if $topcell
-        source($input, $topcell)
-    else
-        source($input)
-    end
-end
-
-if $table_name
-  table_name = $table_name
-else
-  table_name = "main"
+  if $topcell
+    source($input, $topcell)
+  else
+    source($input)
+  end
 end
 
+table_name = $table_name || 'main'
```

Comment:
curious why you don't use the existing global variable here?
ex: `$table_name || = 'main'`

Reply:
Done